tests/engines/registries/integration/components/registries-navbar/component-test.ts

```diff
@@ -301,4 +306,31 @@ module('Registries | Integration | Component | registries-navbar', hooks => {
 
         assert.dom('[data-test-auth-dropdown] ul').isVisible();
     });
+
+    test('branded desktop layout', async function(assert) {
+        const brand = server.create('brand');
+        const brandedProvider = server.create('registration-provider', { name: 'ISPOR', brand });
+
+        this.set('provider', brandedProvider);
+        setBreakpoint('desktop');
+
+        await render(hbs`<RegistriesNavbar @provider={{this.provider}} />`);
+        await percySnapshot(assert);
+
+        assert.dom('[data-test-brand-link]').exists('Branded provider name exists');
+        assert.dom('[data-test-brand-link]').hasText(brandedProvider.name, 'Branded provider name is correct');
+    });
+
+    test('branded name does not show up on mobile layout', async function(assert) {
+        const brand = server.create('brand');
+        const brandedProvider = server.create('registration-provider', { name: 'ISPOR', brand });
+
+        this.set('provider', brandedProvider);
+        setBreakpoint('mobile');
+
+        await render(hbs`<RegistriesNavbar @provider={{this.provider}} />`);
+        await percySnapshot(assert);
+
+        assert.dom('[data-test-brand-link]').doesNotExist('Branded provider name exists');
```

Comment:
Same thing here. I think the test message should be `'Branded provider name does not exist'`